### ISSUE TITLE
[BUG] Defining multiple signatures for method generates incorrect javascript

### ISSUE THREAD

**Issue by SirJosh3917** [alangpierce/sucrase#592]
**To reproduce:**

```ts
export default class Class {
  method(a: number);
  method(a: unknown) {}

  field: number;
}
```

Run the command `yarn sucrase -d out -t typescript in`, and the following output is generated:

```js
export default class Class {
  
  method(a) {}

  field;
}
```

Putting the fields above the method works as a workaround.

**Comment by alangpierce** [comment]
Thanks for reporting! I tracked it down and will try to get a fix published soon.

**Comment by alangpierce** [comment]
Fixed in #593, and I'll do a publish now.
